first announcement draft (#22)
scverse/scverse.github.io: content/blog/hello-world.md

## What are we doing?

Our first priority is to build a strong and lasting community around the tools and the associated analysis. The knowledge to build and maintain our tools should be shared among many to avoid single points of failures. Therefore, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community. We are also hoping that a healthy scverse community will naturally guide us to develop tooling where it is the most required.

There are already a few visible outcomes of our joint work including improved 
[mudata performance](https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2) and
[mudata documentation](https://mudata.readthedocs.io/en/latest/). Additionally, 
[scvi-tools is now using MuData](https://github.com/scverse/scvi-tools/pull/1444) for the representation of multimodal data. 
[Scanpy has gained new workshop notebooks](https://github.com/scverse/scanpy-tutorials/pull/52), and, moreover, we have started 

> [a new page with tutorials for scverse libraries](https://scverse.org/tutorials/).

Reviewer: Add the new template here

Reviewer 2: @giovp should we mention spatial stuff here as well?

## Who is a part of scverse?

scverse is explicitly an effort beyond just a single research group. Only together can we continue to ensure that the ecosystem grows and stays interoperable.
We therefore define several [roles](https://scverse.org/people/) of which only a subset are denoted below.

> [Isaac Virshup](https://github.com/ivirshup), [Danila Bredikhin](https://github.com/gtca) and [Lukas Heumos](https://github.com/Zethson) form the steering committee and also belong to the core developers together with [Adam Gayoso](https://github.com/adamgayoso), [Giovanni Palla](https://github.com/giovp), [Gregor Sturm](https://github.com/grst), and [Ilia Kats](https://github.com/ilia-kats). The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.

Reviewer 2: ```suggestion
[Isaac Virshup](https://github.com/ivirshup), [Danila Bredikhin](https://github.com/gtca) and [Lukas Heumos](https://github.com/Zethson) form the steering committee and also belong to the core team along with [Adam Gayoso](https://github.com/adamgayoso), [Giovanni Palla](https://github.com/giovp), [Gregor Sturm](https://github.com/grst), and [Ilia Kats](https://github.com/ilia-kats).
The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.
While all members of the core team are currently core developers, we envision adding a core community role to the project in the near future.
```

Reviewer 2: Thought adding a mention of community team could also be a good place to mention diversity efforts?

Reviewer 2: Is that bad?

> [Isaac Virshup](https://github.com/ivirshup), [Danila Bredikhin](https://github.com/gtca) and [Lukas Heumos](https://github.com/Zethson), who are also a part of the steering committee, belong to the core developers together with [Adam Gayoso](https://github.com/adamgayoso), [Giovanni Palla](https://github.com/giovp), [Gregor Sturm](https://github.com/grst), and [Ilia Kats](https://github.com/ilia-kats). The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.

Reviewer: ```suggestion
[Isaac Virshup](https://github.com/ivirshup), [Danila Bredikhin](https://github.com/gtca) and [Lukas Heumos](https://github.com/Zethson) form the steering committee and also belong to the core developers together with [Adam Gayoso](https://github.com/adamgayoso), [Giovanni Palla](https://github.com/giovp), [Gregor Sturm](https://github.com/grst), and [Ilia Kats](https://github.com/ilia-kats). The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.
```

+++
title = "Hello world"
date = 2022-04-04T14:28:05+01:00
description = "Introducing scverse"
draft = false
+++

> Due to the increasing reliance of scientific research on complex computational pipelines, foundational software in the life sciences has outgrown individual labs and institutes in terms of its scale and impact. This has motivated us to start something more than a code repository for an exciting new project – a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem. We are therefore very excited to announce **scverse** – a consortium for joint development of major packages in omics data analysis in life sciences.

Reviewer 2: ```suggestion
Due to the increasing reliance of scientific research on complex computational pipelines, foundational software in the life sciences has outgrown individual labs and institutes in terms of its scale and impact.
This has motivated us to start something more than a code repository for an exciting new project – a new consortium to organize and support core (mostly) Python tools for single cell life science research.
We are excited to announce **scverse**.
```

Author: Our slogan is `Foundational tools for omics data in the life sciences` -> I'd prefer tools over software here? The rest of your changes I agree with

Reviewer 2: Main point is that this:

Due to the increasing reliance of scientific research on complex computational pipelines, **`foundational software in the life sciences`** has outgrown individual labs and institutes in terms of its scale and impact. This has motivated us to start something more than a code repository for an exciting new project – a new organisation to maintain and support **`foundational tools in the life sciences`** in the Python ecosystem. We are therefore very excited to announce **scverse** – a consortium for joint development of **`major packages in omics data analysis in life sciences`**.

is too much

Idk that we need to even say the exact thing once.

Author: Ahh, ouch yeah. Too tired to notice. Feel free to fix and commit. You have my sign of approval :)

Reviewer 2: 🙏

## What tools are part of scverse?

Our primary experience lies in the field of computational biology with an emphasis on single-cell omics technologies. This experience has defined our initial scope and focus in the very beginning of our journey. Hence, we start with the two most important data structures for single-cell data in the Python ecosystem, [AnnData](https://github.com/scverse/anndata) for uni-modal data and [MuData](https://github.com/scverse/mudata) for multi-modal data, together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are looking forward to expanding with more major libraries and frameworks which fit our vision soon.

## What are we doing?

> Our first priority is to build a strong and lasting community around the tools and the associated analysis. The knowledge to build and maintain our tools should be shared among many to avoid single points of failures. Therefore, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community. We are also hoping that a healthy scverse community will naturally guide us to develop tooling where it is the most required.

Reviewer 2: ```suggestion
Our first priority is to build a strong and lasting community around the core tools powering single cell analysis today.
The knowledge to build and maintain these tools should be shared among many to avoid single points of failures.
Therefore, we are creating open community channels, creating shared infrastructure to ease package development, and building tooling to make creation of scverse based packages easier.
Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community.
As we grow, we are hoping to recruit more members into governance roles as well as find even more ways to support and connect our community of developers and users.
```

+++
title = "Hello world"
date = 2022-04-04T14:28:05+01:00
description = "Introducing scverse"
draft = false
+++

> With the increasing reliance of scientific research on infrastructural and analytical software, foundational tools in life sciences have outgrown individual labs and institutes in terms of their scale and impact. That has encouraged us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem.

Reviewer 3: ```suggestion
Due to the increasing complexity of scientific research on computational pipelines, foundational software in the life sciences has outgrown individual labs and institutes in terms of its scale and impact. This has motivated us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem.
```

Reviewer 3: I think the second sentence is too casual and can be improved

> Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our initial scope and focus in the very beginning of our journey. We are excited to announce **scverse** – a consortium for joint maintenance of core packages in omics data analysis in life sciences.

Reviewer 3: ```suggestion
Our primary experience lies in the field of computational biology with an emphasis on single-cell omics technologies. This experience has defined our initial scope and focus in the very beginning of our journey. We are excited to announce **scverse** – a consortium for joint maintenance of core packages in omics data analysis in life sciences.
```

Reviewer 3: The third sentence doesn't flow so well after the first two

## What tools are part of scverse?

> We start with the two most important data structures, [AnnData](https://github.com/scverse/anndata) and [MuData](https://github.com/scverse/mudata), together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are looking forward to expanding with more major libraries and frameworks which fit our vision.

Reviewer 3: I would start with some context, also as to why we picked these tools. This paragraph is a bit abrupt IMO

## What are we doing?

There are already a few visible outcomes of our joint work including improved 
[mudata performance](https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2) and
[mudata documentation](https://mudata.readthedocs.io/en/latest/). Additionally, 
[scvi-tools is now using MuData](https://github.com/scverse/scvi-tools/pull/1444) for the representation of multimodal data. 
[Scanpy has gained new workshop notebooks](https://github.com/scverse/scanpy-tutorials/pull/52), and, moreover, we have started 

> [a new page with tutorials for scverse libraries](https://scverse.org/tutorials/).

Reviewer 3: these are all niche. I would think that the largest audience for this post will be people who use our software. Thus, I would start with how the first priority is really building the community, as even what we've done so far is impossible without this. Like I'd start:

1. Our primary goal is to improve how the packages work together etc etc
2. To achieve this we'd like to both build a community of developers, but more importantly users to help us direct the software in the long-term
3. Then I would discuss things we've done in the short term that this has enabled

While we're continuing to work on the scverse libraries and frameworks, we're trying to improve how they work and are tested together. Hence, we are looking into the development of infrastructure which cross tests release candidates of our tools. Further, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Beyond technical challenges we are looking to build a community around scverse, for instance with workshops and hackathons.

## Who is a part of scverse?

scverse is explicitly an effort beyond just a single research group. Only together can we continue to ensure that the ecosystem grows and stays interoperable.

> We therefore define several [roles](https://scverse.org/people/) of which only the most relevant are denoted below.

Reviewer 3: I would replace "most relevant" as it does not come off very well IMO

[Aviv Regev](https://biology.mit.edu/profile/aviv-regev/), [Sarah Teichmann](https://www.sanger.ac.uk/group/teichmann-group/), [Dana Pe'er](https://www.mskcc.org/research/ski/labs/dana-pe-er) and [Bonnie Berger](https://people.csail.mit.edu/bab/) are our scientific advisory board which will help to shape the overall vision and to define the right priorities.

## How can I join and contribute?

We're particularly excited about the community coming together for this effort with so many brilliant people involved who design tools, write code, manage and analyse data and visualise results. Please ask usage questions on [our Discourse forum](https://discourse.scverse.org/), developer questions on [our scverse Zulip](https://scverse.zulipchat.com/) and make sure to [follow us on Twitter](https://twitter.com/scanpy_team) to keep track of scverse updates.

> scverse is very much open to contributions in all areas by **everyone**. We are actively looking to diversify our members and want to especially encourage people from underrepresented backgrounds to contribute and to reach out.

Reviewer 3: Last sentence can be improved

Therefore, we are creating open community channels, creating shared infrastructure to ease package development, and building tooling to make creation of scverse based packages easier.
Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community.
As we grow, we are hoping to recruit more members into governance roles as well as find even more ways to support and connect our community of developers and users.

There are already a few visible outcomes of our joint work including improved 
[mudata performance](https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2) and
[mudata documentation](https://mudata.readthedocs.io/en/latest/). Additionally, 
[scvi-tools is now using MuData](https://github.com/scverse/scvi-tools/pull/1444) for the representation of multimodal data. 
[Scanpy has gained new workshop notebooks](https://github.com/scverse/scanpy-tutorials/pull/52), and, moreover, we have started 
[a collection of learning resources for scverse libraries](https://scverse.org/learn/).

## Who is a part of scverse?

`scverse` is explicitly an effort beyond just a single research group.

> As a community effort we ensure continued interoperablility, promote growth, and increase our bus number. 

Reviewer 2: ```suggestion
As a community effort we ensure continued interoperablility, easier growth, and democratized governance.
```

Reviewer 2: Or "distributed responsibility"
